### DISCUSSION TITLE
One navigateable bufferline through all editors/tabs/splits

### DISCUSSION THREAD

**Discussion by adambasis** [JetBrains/ideavim#740] [Q&A]
In Vim you have a single buffer collection with which you can navigate through:
![image](https://github.com/JetBrains/ideavim/assets/63406944/9b9fec54-8b16-4cf0-8c8e-ca4b40cf3e71)

In Vim you can navigate through all open buffers regardless of in what split you are in. It's not a seperate list of files as any other split. In Intellij, you can only navigate the files on the active tab. So every split has its own set of files. When navigating files in e.g. the left editor tab in Intellij, I can only access the files which are on that tab.
Is it possible to change the behavior so that it is the same as in Vim? (One set of files for all editor tabs). I would really appreciate it, as coming from vim, it really hurts my workflow.

**Comment by citizenmatt** [comment]
You can disable tabs completely in the editor settings. Without the tabs UI, it's a similar experience to Vim showing a single screen split into one or more windows. However, there isn't a way to show extra tab information in an alternative manner, such as across the top of the IDE.

**Reply by adambasis** [reply]
Ahh great! This does exactly what I need!
And I also just realized, that I should use the `:bnext` and `:bprevious` commands in IdeaVim instead of the `<Action>NextTab` etc.

I think I am just going to use the breadcrumbs and put them at the bottom to know which files are in which splits. Thank you!
